testing/src/main/java/org/oppia/android/testing/espresso/TextInputAction.kt

```diff
@@ -60,5 +60,30 @@ class TextInputAction {
         description.appendText("There is no error text.")
       }
     }
+
+    /**
+     * Checks that the TextInputLayout view has the [expectedHelperText] helper text.
+     *
+     * @return a [BoundedMatcher] that matches the [expectedHelperText] with the TextInputLayout's
+     *     helper text
+     */
+    fun hasHelperText(expectedHelperText: String): BoundedMatcher<View, TextInputLayout> {
+      return HelperTextExistsMatcher(expectedHelperText)
+    }
+
+    /**
+     * Class which inherits [BoundedMatcher] and overrides [matchesSafely] function to match the
+     * [expectedHelperText] with the TextInputLayout's helper text.
+     */
+    private class HelperTextExistsMatcher(private val expectedHelperText: String) :
+      BoundedMatcher<View, TextInputLayout>(TextInputLayout::class.java) {
```

Comment:
```suggestion
    private class HelperTextExistsMatcher(
      private val expectedHelperText: String
    ) : BoundedMatcher<View, TextInputLayout>(TextInputLayout::class.java) {
```

Typically we break at the higher syntactic level.

Reply:
This change has also been applied with the latest push. PTAL